tests/constants/admincli/vmdkops.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2017 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// A home to hold test constants related with vmdkops_admin cli.
+
+package admincli
+
+const (
+	// location of the vmdkops binary
+	vmdkops = "/etc/init.d/vmdk-opsd "
+
+	// VmdkopsStatus gives vmdkops service status
+	VmdkopsStatus = vmdkops + "status"
```

Comment:
I'd suggest to give some shorter names such as ServiceStatus, StartService, StopService, RestartService. Please note that we should use "verb+noun" style to keep all constants consistent (e.g. CreateVolume, RemoveVolume, etc.)